### PR TITLE
PR for this Weeks 4 Polls (4/15)

### PR DESCRIPTION
**Pull request by tylerrihn**
@patrick-j-govalpha please review when you get a chance

### CODE REVIEW

File: governance/polls/MIP9 Community Greenlight Poll - HVBANK (Huntingdon Valley Bank Loan Syndication Collateral) - April 18, 2022.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+   - Greenlight
+   - Medium Impact
+options:
+   0: Abstain
+   1: Yes (Greenlight)
+   2: No (Defer)
+start_date: 2022-04-18 16:00:00
+end_date: 2022-05-02 16:00:00
+---
+# Poll: Community Greenlight Poll - Investigate Implementation of HVBANK (Huntingdon Valley Bank Loan Syndication Collateral) - Monday, April 18 at 16:00 UTC
```
Comment (patrick-j-govalpha) [inline governance/polls/MIP9 Community Greenlight Poll - HVBANK (Huntingdon Valley Bank Loan Syndication Collateral) - April 18, 2022.md:17]: This should say "Poll: Community Greenlight Poll - HVBANK (Huntingdon Valley Bank Loan Syndication Collateral) - April 18, 2022"

File: governance/polls/MIP9 Community Greenlight Poll - sETH2 (Stakewise Staked ETH) - April 18, 2022.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+   - Greenlight
+   - Medium Impact
+options:
+   0: Abstain
+   1: Yes (Greenlight)
+   2: No (Defer)
+start_date: 2022-04-18 16:00:00
+end_date: 2022-05-02 16:00:00
+---
+# Poll: Community Greenlight Poll - Investigate Implementation of sETH2 (Stakewise Staked ETH) - Monday, April 18 at 16:00 UTC
```
Comment (patrick-j-govalpha) [inline governance/polls/MIP9 Community Greenlight Poll - sETH2 (Stakewise Staked ETH) - April 18, 2022.md:17]: This should say "Poll: Community Greenlight Poll - sETH2 (Stakewise Staked ETH) - April 18, 2022"

File: governance/polls/MIP9 Community Greenlight Poll - HVBANK (Huntingdon Valley Bank Loan Syndication Collateral) - April 18, 2022.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+vote_type: Plurality Voting
+categories:
+   - Collateral
+   - Greenlight
+   - Medium Impact
+options:
+   0: Abstain
+   1: Yes (Greenlight)
+   2: No (Defer)
+start_date: 2022-04-18 16:00:00
```
Comment (patrick-j-govalpha) [inline governance/polls/MIP9 Community Greenlight Poll - HVBANK (Huntingdon Valley Bank Loan Syndication Collateral) - April 18, 2022.md:14]: Should be: "start_date: 2022-04-18T16:00:00"

File: governance/polls/MIP9 Community Greenlight Poll - HVBANK (Huntingdon Valley Bank Loan Syndication Collateral) - April 18, 2022.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+end_date: 2022-05-02 16:00:00
```
Comment (patrick-j-govalpha) [inline governance/polls/MIP9 Community Greenlight Poll - HVBANK (Huntingdon Valley Bank Loan Syndication Collateral) - April 18, 2022.md:15]: should be: "end_date: 2022-05-02T16:00:00"

File: governance/polls/MIP9 Community Greenlight Poll - sETH2 (Stakewise Staked ETH) - April 18, 2022.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+categories:
+   - Collateral
+   - Greenlight
+   - Medium Impact
+options:
+   0: Abstain
+   1: Yes (Greenlight)
+   2: No (Defer)
+start_date: 2022-04-18 16:00:00
+end_date: 2022-05-02 16:00:00
```
Comment (patrick-j-govalpha) [inline governance/polls/MIP9 Community Greenlight Poll - sETH2 (Stakewise Staked ETH) - April 18, 2022.md:15]: should be: "end_date: 2022-05-02T16:00:00"

File: governance/polls/MIP9 Community Greenlight Poll - sETH2 (Stakewise Staked ETH) - April 18, 2022.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+vote_type: Plurality Voting
+categories:
+   - Collateral
+   - Greenlight
+   - Medium Impact
+options:
+   0: Abstain
+   1: Yes (Greenlight)
+   2: No (Defer)
+start_date: 2022-04-18 16:00:00
```
Comment (patrick-j-govalpha) [inline governance/polls/MIP9 Community Greenlight Poll - sETH2 (Stakewise Staked ETH) - April 18, 2022.md:14]: Should be: "start_date: 2022-04-18T16:00:00"

**Review by patrick-j-govalpha — CHANGES_REQUESTED** [review]
Please see comments

**Comment by tylerrihn** [comment]
Deployed those changes @patrick-j-govalpha  please look over again

File: governance/polls/MIP9 Community Greenlight Poll - sETH2 (Stakewise Staked ETH) - April 18, 2022.md [hunk 7]
```diff
@@ -11,10 +11,10 @@ options:
    0: Abstain
    1: Yes (Greenlight)
    2: No (Defer)
-start_date: 2022-04-18 16:00:00
-end_date: 2022-05-02 16:00:00
+start_date: 2022-04-18T16:00:00
+end_date: 2022-05-02T16:00:00
 ---
-# Poll: Community Greenlight Poll - Investigate Implementation of sETH2 (Stakewise Staked ETH) - Monday, April 18 at 16:00 UTC
+# Poll: Community Greenlight Poll - HVBANK (Huntingdon Valley Bank Loan Syndication Collateral) - April 18, 2022
```
Comment (patrick-j-govalpha) [inline governance/polls/MIP9 Community Greenlight Poll - sETH2 (Stakewise Staked ETH) - April 18, 2022.md:17]: sETH2 not HVBANK

**Review by patrick-j-govalpha — CHANGES_REQUESTED** [review]

**Comment by tylerrihn** [comment]
Ok please review YET AGAIN (sorry) @patrick-j-govalpha

File: governance/polls/MIP9 Community Greenlight Poll - sETH2 (Stakewise Staked ETH) - April 18, 2022.md [hunk 8]
```diff
@@ -14,7 +14,7 @@ options:
 start_date: 2022-04-18T16:00:00
 end_date: 2022-05-02T16:00:00
 ---
-# Poll: Community Greenlight Poll - HVBANK (Huntingdon Valley Bank Loan Syndication Collateral) - April 18, 2022
+# Poll: Investigate Implementation of sETH2 (Stakewise Staked ETH) - Monday, April 18 at 16:00 UTC
```
Comment (patrick-j-govalpha) [inline governance/polls/MIP9 Community Greenlight Poll - sETH2 (Stakewise Staked ETH) - April 18, 2022.md:17]: This should say "Poll: Community Greenlight Poll - sETH2 (Stakewise Staked ETH) - April 18, 2022"

**Review by patrick-j-govalpha — CHANGES_REQUESTED** [review]

**Review by patrick-j-govalpha — APPROVED** [review]
LGTM, thanks Tyler. @LongForWisdom

**Pull request merged**